recipe/build.sh

```diff
@@ -294,6 +294,7 @@ if [[ ${HOST} =~ .*darwin.* ]]; then
                 -no-egl \
                 -no-openssl \
                 -optimize-size \
+                -no-use-gold-linker \
```

Comment:
Gold linker thing is not the error

Reply:
sorry, brain fart.

Should we try to add `-Welaborated-enum-base` or `-Wno-elaborated-enum-base`